easybuild/easyconfigs/i/intltool/intltool-0.51.0-GCCcore-6.4.0-Perl-5.26.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'intltool'
+version = '0.51.0'
+versionsuffix = '-Perl-5.26.0'
+
+homepage = 'http://freedesktop.org/wiki/Software/intltool/'
+description = """intltool is a set of tools to centralize translation of
+ many different file formats using GNU gettext-compatible PO files."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://launchpad.net/intltool/trunk/%(version)s/+download/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('XML-Parser', '2.44_01', versionsuffix),
+]
+
+# intltool bug with Perl 5.26
+# https://bugs.launchpad.net/intltool/+bug/1696658
+# This patch ist from:
+# https://github.com/Alexpux/MSYS2-packages/blob/master/intltool/perl-5.22-compatibility.patch
+patches = ['intltool-perl-5.26.0-fix-regex.patch']
```

Comment:
@reedts Since the patch is in no way specific to Perl 5.26.0, I would avoid including a version number in the patch filename?
Do include the `intltool` version though, since the patch *is* specific to this version of `intltool`.

So, I'd suggest: `intltool-%(version)s-Perl-fix-regex.patch`.

Reply:
My bad. The problem seems to be specific to the combination of `intltool-0.51.0` and `Perl-5.26.0` but the patch file itself is not specific to this combination. I will fix that.